vendor/modules.txt

```diff
@@ -355,8 +355,8 @@ go.uber.org/ratelimit
 golang.org/x/mod/internal/lazyregexp
 golang.org/x/mod/module
 golang.org/x/mod/semver
-# golang.org/x/net v0.33.0
-## explicit; go 1.18
+# golang.org/x/net v0.36.0
+## explicit; go 1.23
```

Comment:
uhm, I just spotted this.
Problem is: we build with golang 1.19
https://github.com/openshift-kni/numaresources-operator/blob/release-4.13/Dockerfile.tools#L5

Reply:
Thanks for point this. Since we don't want to bump golang versions other than the z versions and this is not a flagged CVE we can close this.

/close